requirements.txt

```diff
@@ -1,6 +1,6 @@
 awscli==1.19.14
 biopython==1.78
-ipython==7.16.1
+ipython==7.16.3
```

Comment:
I wanted to test the change to these requirements, but as is they do not install (not even as is on `master`).  

There are transitive boto3 deps that cannot be satisfied.

```
There are incompatible versions in the resolved dependencies:
  botocore<1.24.0,>=1.23.41 (from boto3==1.20.41->toil[aws,cwl,encryption,google,mesos]==5.2.0->-r requirements.in (line 7))
  botocore<2.0a.0,>=1.12.36 (from s3transfer==0.3.7->awscli==1.19.14->-r requirements.in (line 1))
  botocore==1.20.14 (from awscli==1.19.14->-r requirements.in (line 1))
```

I recommend using pip-tools to manage both the core and downstream python dependencies.  I will open a new PR for that.